conceptnet5/readers/emoji.py

```diff
@@ -0,0 +1,49 @@
+from conceptnet5.edges import make_edge
+from conceptnet5.formats.msgpack_stream import MsgpackStreamWriter
+from conceptnet5.uri import Licenses
+from conceptnet5.nodes import standardized_concept_uri
+
+import xml.etree.ElementTree as ET 
+
+
+def is_sentence(text):
+    """
+    There are a few instances where a sentence of
+    multiple words is used to describe an emoji
+    (which is not very helpful in our case) AS WELL AS
+    single words or phrases, which are separated by '|'.
+    Using this function, we can ignore the sentences/phrases,
+    and only look at single words, which is better for
+    conceptnet to handle.
+    """
+    return (' ' in text and '|' not in text)
+
+
+def strip_words(text):
+    """
+    When multiple words (not in a sentence, but
+    separated by '|') are used to describe emojis,
+    we need to remove the '|' in order to create
+    edges for each word. This function takes out
+    the '|' and puts all the words into a list.
+    """
+    return text.split(' | ')
+
+
+def handle_file(input_file, output_file):
+    tree = ET.parse(input_file)
+    out = MsgpackStreamWriter(output_file)
+    root = tree.getroot()
+    lang = root[0][1].attrib['type']
+    for annotation in root[1]:
+    	if not is_sentence(annotation.text):
+            rel = '/r/symbolOf'
+            start = standardized_concept_uri('mul', annotation.attrib['cp'])
+            license = Licenses.cc_attribution
+            sources = [{'contributor': '/s/contributor/cldr/31'}]
```

Comment:
While the dictionary key here should remain as 'contributor' to match the structure of other data, the URI we use for sources can be more specific and doesn't have to call the Unicode CLDR a "contributor". The URI I suggested is `/s/resource/unicode/cldr/31`, making it similar to the WordNet source defined on line 12 of wordnet.py.

Also, this dictionary is a constant. Instead of having to construct it every time we go through the loop, you could define it once outside the function, near the top of the file, with a variable name in all capital letters to indicate that it's a constant. This is also similar to wordnet.py.